tests/benches/collect_and_decode.rs

```diff
@@ -27,7 +27,7 @@ fn compile_runner(tempdir: &TempDir) -> PathBuf {
     exe.push(tempdir);
     exe.push(src.file_stem().unwrap());
 
-    let mut compiler = mk_compiler(&ykllvm_bin("clang"), &exe, &src, &[], false);
+    let mut compiler = mk_compiler(&ykllvm_bin("clang"), &exe, &src, &[], false, None);
```

Comment:
I might be wrong but this might need `cargo fmt`.

Reply:
Just tried locally. Seems OK. Line is 87 chars long.